docs/logs/flattener-processor.md

```diff
@@ -0,0 +1,32 @@
+title: Flattener Processor
+description: Used to flatten a nested structure into separate fields
+
+This processor enables the transformation of nested JSON array objects into individual fields. Once segregated, these fields can be renamed and utilized as metrics, enabling the creation of charts and data visualization.
```

Comment:
Maybe something here should be linked to https://sematext.com/docs/logs/logs-table-quick-actions/ ?

Reply:
We don't offer flattener from the quick actions though.

Reply:
Aha, OK